trustpinning/certs.go

```diff
@@ -86,7 +86,7 @@ We shall call this: TOFUS.
 
 Validation failure at any step will result in an ErrValidationFailed error.
 */
-func ValidateRoot(certStore trustmanager.X509Store, root *data.Signed, gun string, trustPinning TrustPinConfig) error {
+func ValidateRoot(prevRoot *data.SignedRoot, root *data.Signed, gun string, trustPinning TrustPinConfig) error {
```

Comment:
Non-blocking: Wondering if it'd make sense to have ValidateRoot return the completely unmarshalled root?  `NotaryRepository.bootstrapClient`, for instance, calls `data.RootFromSigned` anyway after ValidateRoot succeeds.  This would also be useful for the builder.